rwandaemr-installer/src/main/resources/scripts/create-service-user.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+## load variables from the vars.txt file
+source vars.txt
+
+home_dir=${parent_dir}
+
+if [ "$env_type" == "demo" -o "$env_type" == "prod" -o "$env_type" == "test" ]; then
+  echo "Checking if user doesn't exist"
+  if [ $(id -u) -eq 0 ]; then
+	egrep "^$username" /etc/passwd >/dev/null
+	if [ $? -eq 0 ]; then
+		echo "$username exists! Skipping!"
+		exit 1
```

Comment:
Do you really want to exit 1 here?  Won't this cause the whole process to fail?  I think we'd want this to work similarly to Ansible / Puppet - be able to run it over and over and have it ensure the environment is in a certain state.

Reply:
This is the message you get when the user exists
`Checking if user doesn't exist
xxxtest exists! Skipping!
`
If username exists then it should exist but this only exits the create_service_user() only.

Reply:
Sounds good, I just wanted to point it out.  If it works correctly, great.